stubs/pdo.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+class PdoStatement {
+    /**
+     * @template T
+     * @param class-string<T> $class
+     * @return false|T
+     */
+    public function fetchObject($class = "stdclass") {}
```

Comment:
@muglug should parameters like this one, which make the php engine create a object based on a userland class-name, be considered a taint-sink?

Reply:
Just went ahead and proposed a change: https://github.com/vimeo/psalm/pull/4701